### PR TITLE
Metrica: retry query if quota exceeded

### PR DESCRIPTION
**Pull request by denisov-vlad**
## What type of PR is this? (check all applicable)

- [ ] Feature

## Description

Metrica has a limit for simultaneous queries. If you have dashboard with 5+ different queries there is always errors. I use backoff library to retry queries if it fails with 429 (quota exceeded) error.

### CODE REVIEW

**Comment by denisov-vlad** [comment]
@getredash/mergers hi! could you check this request please?

**Comment by justinclift** [comment]
@denisov-vlad Would you be ok to resolve the merge conflict? :smile:

File: requirements.txt [hunk 1]
```diff
@@ -55,6 +55,10 @@ sshtunnel==0.1.5
 supervisor==4.1.0
 supervisor_checks==0.8.1
 werkzeug==2.3.6
+backoff==1.11.1
+# Install the dependencies of the bin/bundle-extensions script here.
+# It has its own requirements file to simplify the frontend client build process
+-r requirements_bundles.txt
```
Comment (justinclift) [inline requirements.txt:61]: This seems to be causing an error:

```
ERROR: Could not open requirements file: [Errno 2] No such file or directory:
'requirements_bundles.txt'
```

**Comment by denisov-vlad** [comment]
Oops, something went wrong with this PR. I will recreate without these errors.

**Comment by justinclift** [comment]
No worries. :smile:

**Pull request closed without merging**